connector/src/main/java/org/geysermc/connector/network/translators/item/translators/BannerTranslator.java

```diff
@@ -87,6 +86,11 @@ public ItemStack translateToJava(ItemData itemData, ItemEntry itemEntry) {
 
     @Override
     public List<ItemEntry> getAppliedItems() {
+        if (appliedItems == null) {
+            appliedItems = GeyserEdition.TOOLBOX.getItemEntries().values().stream()
+                    .filter(entry -> entry.getJavaIdentifier().endsWith("banner"))
+                    .collect(Collectors.toList());
+        }
```

Comment:
This should stay how it was so we are not searching through the items every time its called, it should just bee cached as it shouldn't change.

Reply:
Its not searched each time. You'll note that it checks if appliedItems is null and if not does the search a single time and caches it from then onwards.